app/views/times/new.html.erb

```diff
@@ -1,5 +1,49 @@
-<h1>New Time</h1>
+<%= render 'navbar' %>
+
+<div class="jumbotron" id="billboard">
+  <div class="dates-left-side">
+    <h1>Choose Dates</h1>
+    
+    <% if flash[:message] %>
+       <div class="alert alert-danger alert-dismissible show" role="alert">
+        <div class="error"><strong>Error: </strong><%= flash[:message] %></div>
+          <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+      </div>
+    <% end %>
+    
+    <%= form_for(@time) do |f| %>
+      <% if @time.errors.any? %>
+        <div id="error_explanation">
+          <h2><%= pluralize(@time.errors.count, "error") %> prohibited this time from being saved:</h2>
+    
+          <ul>
+          <% @time.errors.full_messages.each do |message| %>
+            <li><%= message %></li>
+          <% end %>
+          </ul>
+        </div>
+      <% else %>
+        <div class="form-group">
+            <%= f.text_field :date_time, placeholder: "Date(s)", :autocomplete => :off, class: "datepicker"%>
```

Comment:
Perhaps change this text field into a button instead? Users would always pick their dates with the datepicker UI instead of typing in dates, so putting a date field seems quite clunky.

Reply:
I think theres a fix for the datepicker so that entering input manually and pressing enter works, ill check it out.